package.json

```diff
@@ -7,6 +7,7 @@
     "url": "git@github.com:raml-org/raml-js-parser.git"
   },
   "main": "./lib/raml.js",
+  "bin": "bin/raml-parser.js",
```

Comment:
But you don't run it like that. It takes the name from the package. Or from an object you pass in there. But I can remove it.